### PR TITLE
Add Default API Alerts

### PR DESCRIPTION
**Pull request by joshw123**
---
**Checklist**

I applicable, I acknowledge that I have:
- [X] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [X] Ran `terraform fmt` on all modified files
- [X] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [X] Made sure all relevant tests pass

### CODE REVIEW

File: modules/project/README.md [hunk 1]
```diff
@@ -1360,12 +1360,50 @@ module "bucket" {
...
+module "project" {
+  source          = "./fabric/modules/project"
+  billing_account = var.billing_account_id
+  name            = "project"
+  parent          = var.folder_id
+  prefix          = var.prefix
+  services = [
+    "stackdriver.googleapis.com"
+  ]
+  enable_api_alerts = true
```
Comment (ludoo) [inline modules/project/README.md:1396]: can we combine both into a single `api_alerts` variable, with `.enabled` and `default_email` attributes?

**Review by ludoo — COMMENTED** [review]
I'm not sure this should be in the project module: while this is useful, it's not a product feature but a "recipe" or collection of "recipes" which we typically never embed in modules. Plus, having them in code makes it very hard for users to change/tweak them.

At a minimum, recipes should be defined via YAML files in a factory. But I still don't like them in the base project module. What if we added support for it in the project factory, which is where you would typically enable/disable alerts?

**Comment by joshw123** [comment]
> I'm not sure this should be in the project module: while this is useful, it's not a product feature but a "recipe" or collection of "recipes" which we typically never embed in modules. Plus, having them in code makes it very hard for users to change/tweak them.
> 
> At a minimum, recipes should be defined via YAML files in a factory. But I still don't like them in the base project module. What if we added support for it in the project factory, which is where you would typically enable/disable alerts?

Hey, Yeah I can look at putting it within the project-factory, The only reason I didn't do that was with compliance checks, these alerts are required in every project, and project-factory is only used in stage-2 project factory I believe? @ludoo

**Comment by ludoo** [comment]
Wait :) Let's leave this open and have @juliocc and @wiktorn also comment :)

**Comment by ludoo** [comment]
reopening

**Comment by juliocc** [comment]
Agree with @ludoo, this shouldn't be part of the project module.

**Pull request closed without merging**